### ISSUE TITLE
A11y_Visual Studio Code Client_Sep2021_Debug Adapter Protocol_Overview_ColorContrastAnalyzer: Color contrast ratio of links with respect to its surrounding text is less than 3:1.(Given 1.9:1)

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/debug-adapter-protocol#211]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTag:**#A11yTCS;#DesktopWeb;#Win10;#WCAG1.4.11;#FTP;#A11yMAS;#ChromiumEdge;#A11ySev2;#P2_Visual Studio Code Clien_Sep2021;#ColorContrastAnalyzer;#Visual Studio Code Client;#A11yWCAG2.1;#P2_WCP;

## **Environment Details:**

Application Name: Visual Studio Code Client
#URL: https://microsoft.github.io/debug-adapter-protocol
Browser Versions: Microsoft Edge Version 93.0.961.38 (Official build) (64-bit)
Windows Version: Windows10

**Example Page URL**: https://microsoft.github.io/debug-adapter-protocol/overview

## **Repro Steps:**

1. Open the URL: https://microsoft.github.io/debug-adapter-protocol/specification
2. Debug Adapter Protocol Overview screen will get open
3. Open Color contrast analyzer tool and check the contrast ratio of links with respect to its surrounding text .

## **Actual:**

Color contrast ratio of links with respect to its surrounding text is less than 3:1.(Given 1.9.:1)

**Same issue is also observed with throughout the application.**

## **Expected:**

Contrast ratio of all the links with respect to its surrounding text should be more than or equal to 3:1.

## **User Impact:**

Low vision users will not be able to see links between surrounding text if contrast ratio is less.

## **Attachment:**
![#211](https://user-images.githubusercontent.com/60422426/133768506-e90a0fe9-be43-4a37-8a7e-60af954eac74.gif)

**Comment by sana-ajani** [comment]
The ratio of the link color with the white background is above 3:1. Based on this [WCAG rule]( https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html), it is comparing the text color with the background, not the color of the other foreground text.

**Comment by NiharikaTipirishetty** [comment]
#P2_debugadapterprotocol_Web_Sep2021;

**Comment by prabhudevu** [comment]
I have verified this issue in the https://microsoft.github.io/debug-adapter-protocol
we are not able to find the same screen that is mentioned in the bug, and we verified the issue for other links in the web paget, it is passing the color contrast ratio. added attachment for reference

![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/98821792/178419513-33a2ab66-416f-438a-b16e-ea06f8b03c63.png)

**Comment by prabhudevu** [comment]
#closed;
